.github/workflows/lint-test.yaml

```diff
@@ -7,11 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
 
       - name: Run chart-testing (lint)
         id: lint
-        uses: helm/chart-testing-action@v1.0.0-alpha.3
+        uses: helm/chart-testing-action@v1.0.0-rc.1
```

Comment:
thanks for that!

I have in my backlog review that and open a PR this week :)

Reply:
Welcome!

I missed that I need to do it here AND in the install line the first time.

Will update the PR description.